src/common/config.cc

```diff
@@ -628,11 +661,41 @@ void Config::UpdateDlMCS(const json& dl_mcs) {
   dl_mod_order_ = static_cast<size_t>(pow(2, dl_mod_order_bits_));
   InitModulationTable(this->dl_mod_table_, dl_mod_order_);
 
+  double dl_code_rate = dl_mcs.value("code_rate", 0.333);
+  if (dl_code_rate >= 1 || dl_code_rate <= 0)
+    std::cerr << "Invalid DL code rate.. Must be in range (0, 1)." << std::endl;
   uint16_t base_graph = dl_mcs.value("base_graph", 1);
-  uint16_t zc = dl_mcs.value("Zc", 68);
+  //uint16_t zc = dl_mcs.value("Zc", 68);
+  //size_t num_rows = dl_mcs.value("nRows", (base_graph == 1) ? 46 : 42);
+  std::vector<size_t> zc_vec = {
+      2,   4,   8,   16, 32, 64,  128, 256, 3,   6,   12,  24, 48,
+      96,  192, 384, 5,  10, 20,  40,  80,  160, 320, 7,   14, 28,
+      56,  112, 224, 9,  18, 36,  72,  144, 288, 11,  22,  44, 88,
+      176, 352, 13,  26, 52, 104, 208, 15,  30,  60,  120, 240};
+  std::sort(zc_vec.begin(), zc_vec.end());
+  size_t zc = SIZE_MAX;
+  size_t i = 0;
+  for (; i < zc_vec.size() - 1; i++) {
+    if (zc_vec.at(i) > 256) {
+      // According to cyclic_shift.cc cyclic shifter for zc
+      // larger than 256 has not been implemented, so we skip them here.
```

Comment:
Go ahead and add  RtAssert(false, "larger than 256 has not been implemented.  Please adjust your configuration");

Reply:
This would not be necessarily a bad thing to happen. I was thinking we could print a warning. Would the following be a good way to do it?
`    MLPD_WARN(
        "Exceeded possible range of LDPC lifting Zc! Setting max possible "
        "lifting size (%zu).\nFor better use of resources, you may reduce your "
        "coding or modulation rate.",
        kMaxSupportedZc);
`

Reply:
Works for me